public/app/features/dashboard-scene/conditional-rendering/ConditionalRenderingVariable.tsx

```diff
@@ -0,0 +1,124 @@
+import { css } from '@emotion/css';
+import { ReactNode, useMemo } from 'react';
+
+import { SceneComponentProps, sceneGraph } from '@grafana/scenes';
+import { Combobox, ComboboxOption, Field, Input, Stack, useStyles2 } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+
+import { ConditionHeader } from './ConditionHeader';
+import { ConditionalRenderingBase, ConditionalRenderingBaseState } from './ConditionalRenderingBase';
+import { handleDeleteNonGroupCondition } from './shared';
+
+export type VariableConditionValue = {
+  name: string;
+  operator: '=' | '!=';
+  value: string;
+};
+
+interface ConditionalRenderingVariableState extends ConditionalRenderingBaseState<VariableConditionValue> {}
+
+export class ConditionalRenderingVariable extends ConditionalRenderingBase<ConditionalRenderingVariableState> {
+  public get title(): string {
+    return t('dashboard.conditional-rendering.variable.label', 'Variable');
+  }
+
+  public evaluate(): boolean {
+    if (!this.state.value.name) {
+      return true;
+    }
+    const variable = sceneGraph.getVariables(this).state.variables.find((v) => v.state.name === this.state.value.name);
+
+    // name is defined but no variable found - return false
+    if (!variable) {
+      return false;
+    }
+    const value = variable.getValue();
+
+    let hit: boolean;
```

Comment:
```suggestion
    let hit = Array.isArray(value) ? value.includes(this.state.value.value) : value === this.state.value.value
```

^^ doing so will avoid the unnecessary if below